error.php

```diff
@@ -21,11 +21,14 @@
     $tpl->setContent($local_tpl->get());
     $tpl->printToStdout();
 } catch (Exception $e) {
+    header("HTTP/1.1 500 Internal Server Error");
```

Comment:
Mind the effect of putting this _here_ (see the next lines about `DEVMODE`).

Reply:
… thinking about it again, won’t be an issue in practice, as user agents will normally also display the body of HTTP 500 responses. Sorry. Should still work in dev mode.